app/app.cpp

```diff
@@ -1,11 +1,56 @@
 #include "app.h"
-#include <algorithm>
 
 namespace vsite::oop::v2
 {
-/*
-* class member function implementations
-*/
+	color::color(double red, double green, double blue) 
```

Comment:
Ako ste već dodali konstruktor (nije dio ove vježbe), onda inicijalizirajte članove prije tijela funkcije
```
color::color(double red, double green, double blue) : m_red(red), m_green(green), m_blue(blue)
{
}
```